helm/dagger/templates/engine-statefulset.yaml

```diff
@@ -99,27 +125,63 @@ spec:
           volumeMounts:
             - name: data
               mountPath: /var/lib/dagger
+            {{- if .Values.engine.storage.run.hostPath.enabled }}
             - name: run
               mountPath: /var/run/buildkit
+            {{- end }}
             {{- if .Values.engine.config }}
             - name: config
               mountPath: /etc/dagger/engine.toml
               subPath: engine.toml
             {{- end }}
+            {{- if .Values.engine.configJson }}
+            - name: config
+              mountPath: /etc/dagger/engine.json
+              subPath: engine.json
+            {{- end }}
+            {{- with .Values.engine.volumeMounts }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
       terminationGracePeriodSeconds: {{ .Values.engine.terminationGracePeriodSeconds }}
       volumes:
+        {{- if not .Values.engine.storage.data.persistentVolumeClaim.enabled }}
```

Comment:
Should this be a `.Values.engine.storage.run.hostPath.enabled` condition instead? We are using a `hostPath` value which will make the connection easier to make.

Reply:
Definitely, nice catch!